TravelAgency-persistence/src/main/java/cz/muni/fi/travelAgency/entities/Trip.java

```diff
@@ -62,7 +62,7 @@
     /**
      * Excursions assigned to trip.
      */
-    @OneToMany(fetch = FetchType.LAZY, mappedBy = "trip", cascade = CascadeType.ALL)
+    @OneToMany(fetch = FetchType.LAZY, mappedBy = "trip", cascade = CascadeType.MERGE)
```

Comment:
This was here to also update on delete -> if trip is deleted, delete it's excursions. Why did you alter it to merge ?

Reply:
Because it could not save detach entity.

Reply:
So you want to prevent it from persisting an entity ? I do not understand clearly.